samples/manifests/minimal/dev-env.yaml

```diff
@@ -8,8 +8,6 @@ UserPoolId: !SSM ${/orbit-foundation/dev-env/resources::UserPoolId}
 SharedEfsFsId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsFsId}
 SharedEfsSgId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
-CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
```

Comment:
don't do this. Ramon will fix this properly

Reply:
Added cognito attributes to manifest files.